drivers/hmis/app/controllers/hmis/graphql_controller.rb

```diff
@@ -100,5 +111,22 @@ def handle_graphql_exception(err)
         data: {},
       }
     end
+
+    def graphql_activity_log(gql_param)
+      Hmis::ActivityLog.new(
+        user_id: current_hmis_user.id, # FIXME: true_user if masquerading
+        data_source_id: current_hmis_user.hmis_data_source_id,
+        ip_address: request.remote_ip,
```

Comment:
I'm a little suspicious that this may not be the actual client IP given that this goes through a proxy. Definitely should check this in production